android/src/main/java/com/bitmovin/player/reactnative/converter/JsonConverter.kt

```diff
@@ -1169,6 +1170,59 @@ class JsonConverter {
             playerViewConfig = toPlayerViewConfig(json),
             pictureInPictureConfig = toPictureInPictureConfig(json.getMap("pictureInPictureConfig")),
         )
+
+        @JvmStatic
+        fun fromBufferLevel(bufferLevel: BufferLevel?): WritableMap? {
+            if (bufferLevel == null) {
+                return null
+            }
+
+            return Arguments.createMap().apply {
+                putDouble("level", bufferLevel.level)
+                putDouble("targetLevel", bufferLevel.targetLevel)
+                putInt(
+                    "media",
+                    when (bufferLevel.media) {
+                        MediaType.Audio -> 0
+                        MediaType.Video -> 1
+                    },
+                )
+                putInt(
+                    "type",
+                    when (bufferLevel.type) {
+                        BufferType.ForwardDuration -> 0
+                        BufferType.BackwardDuration -> 1
+                    },
+                )
+            }
+        }
+
+        @JvmStatic
+        fun fromRNBufferLevels(bufferLevels: RNBufferLevels?): WritableMap? {
```

Comment:
When can bufferLevels be null?

Reply:
Address in 245d413e0f81d2455d76205f22b400731cc7c551  for Android
Address in a0b00ce2ef28efaa316700049bcb3b3671b464d7 for iOS